### ISSUE TITLE
Using online builder - TypeError: a.clone is not a function

### ISSUE THREAD

**Issue by venuziano** [ckeditor/ckeditor5-react#315]
I followed the [official ](https://ckeditor.com/docs/ckeditor5/latest/installation/getting-started/frameworks/react.html#integrating-a-build-from-the-online-builder) guide line using the online builder and I'm getting the error `TypeError: a.clone is not a function`.

Steps to reproduce:

1. Unzip the .rar and moved the files to the root of the repository:
![image](https://user-images.githubusercontent.com/15386984/178842812-b29977e9-cd0d-4132-aa2e-a595d12fb680.png)

2. Executed npm add file:./ckeditor5 or npm install /ckeditor5, both have the same result (I'm not using yarn):
![image](https://user-images.githubusercontent.com/15386984/178842920-98651c55-5bbe-43bf-9d9c-d19746248c12.png)

3. Created a component:
![image](https://user-images.githubusercontent.com/15386984/178843848-78762dcf-78ac-4a4e-982c-64219e9d3f14.png)

5. Imported this component whatever I need to use. I tried to import using two approaches:
```
const TextEditor = dynamic(
  () => import('@/shared/components/CKEditor/CustomCKEditor'),
  {
    ssr: false,
  },
)
```

or simply: 
`import TextEditor from '@/shared/components/CKEditor/CustomCKEditor'`

When I access the page I got this error:
![image](https://user-images.githubusercontent.com/15386984/178843221-a8e01093-fa42-4306-916b-372a67652c03.png)

Environment:
Windows 10
Using React (17.0.2) + Next.js (12.1.0)
Node version: v14.19.0

My babel.config.js file:
```
module.exports = {
  presets: ["next/babel"],
  plugins: [
    ["styled-components", { "ssr": true }],
    ['import', { libraryName: 'antd', style: true }],
    "inline-react-svg"
  ]
}
```

My next.config.js file (I'm using Ant Design library):
```
/** @type {import('next').NextConfig} */
require('dotenv').config();

const withAntdLess = require('next-plugin-antd-less');

module.exports = withAntdLess({
  lessVarsFilePath: './src/styles/ant/antd.less',
  modifyVars: {
    '@primary-color': '#009BCA',
    '@border-radius-base': '10px',
    '@btn-border-radius-base': '20px',
    '@label-color': '#000000',
    '@form-item-label-font-size': '16px',
    '@form-item-trailing-colon': 'false',
    '@form-item-margin-bottom': '12px',
    '@checkbox-border-radius': '2px',
    '@dropdown-selected-color': '#F5F5F5',
   },
  lessVarsFilePathAppendToEndOfContent: false,
  nextjs: {
    localIdentNameFollowDev: true,
  },
  esModule: false,

  webpack(config) {
    return config;
  },
})
```

What Am I doing wrong here? I see many people managing to work using the online builder. I have to be doing something different.

**Comment by venuziano** [comment]
**Just a quick note**: When I run `npm add file:./ckeditor5` or `npm install`, should the `ckeditor `root file have a `node_modules` folder?

**Comment by venuziano** [comment]
I installed yarn to give a try. Reproduced the same steps above, now I got this error:

![image](https://user-images.githubusercontent.com/15386984/178847280-3f994d07-99d9-40a3-b062-a52d28c03e85.png)

**Comment by dufipl** [comment]
Hello @venuziano, thank you for the report.

I was following our official guide for integrating React components with CKEditor 5 online build and unfortunately was not able to reproduce your issue. However, please be aware that I was testing our official path from the guide on example React app, I was not looking at your specific project or configs, because it is not impossible for us to reproduce issues within your project. 

I would recommend you to try to integrate React component with an example project created with `npx create-react-app ckeditor5-react-example` and integrate CKEditor 5 there, as I did. If that works, the problem is probably with your project config, and you might want to look for eventual mistakes.

I hope that helps, but if you have any additional questions, please feel free to ask.

**Comment by venuziano** [comment]
Thanks for answering me @dufipl! I will try your suggestion.

**Comment by venuziano** [comment]
Passing here to give the steps I did to CKEditor work with Ant design + Next.js

1. yarn add @ckeditor/ckeditor5-react @ckeditor/ckeditor5-build-classic
2. yarn add @ckeditor/ckeditor5-build-classic (I'm using de classic build)
3. yarn add file:./ckeditor5
4. I had conflicts with another npm package called antd-mask-input, it gave me the error this.useState is not function, so I had to use another package for masked input. With react-input-mask it worked.
5. To import the ckeditor component I had to use dynamic plus ssr false, like this:
`const TextEditor = dynamic(
  () => import('@/shared/components/CKEditor/CustomCKEditor'),
  {
    ssr: false,
  },
)`

Otherwise I got an error that window was undefined.

And with this all worked fine.
